config/recipes/istio-gateway/03-elasticsearch-kibana.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: ekmnt
+  namespace: istio-apps
+  labels:
+    app: ekmnt
+spec:
+  version: 7.6.0
+  http:
+    tls:
+      selfSignedCertificate:
+        disabled: true
+  nodeSets:
+  - name: master
+    count: 1
+    config:
+      node.master: true
+      node.data: false
+      node.ingest: false
+      node.ml: false
+      node.store.allow_mmap: false
```

Comment:
I wonder if we should not use this setting everywhere. We added it to the quickstart to avoid that users have a bad experience when they try out ECK for the first time, but I am bit concerned that we are adding it everywhere now without comment.

Reply:
I added a README to the root of the recipes directory as you suggested and included a warning on it about using this setting. Do you think that's sufficient?

I could get rid of the setting from the manifests but that could increase the likelihood of these recipes not working out of the box -- leading to a bad user experience.

Reply:
👍